scripts/symbiotic

```diff
@@ -222,18 +224,23 @@ def set_environ(tool, is32bit = False):
     prepend('C_INCLUDE_DIR', '{0}/include'.format(symbiotic_dir))
     prepend('C_INCLUDE_DIR', '{0}/llvm-{1}/include'.format(symbiotic_dir, llvm_version))
 
-    assert tool.name() == 'klee'
+    #assert tool.name() == 'klee'
 
     # override the path completely, we want to be sure that we use only our binaries
+    if additional_path:
```

Comment:
CPAchecker requires Java, and thus a default `PATH`.
I think it would be best to simply prepend your `PATH` elements to the environments `PATH` - then, the binaries of symbiotic are used, and if a binary can't be found there, a default one is used. But since this changes the default behavior of symbiotic and I didn't know whether this would be ok with you, I instead introduced this new option.

Reply:
The reason why the `PATH` variable was overridden was to be sure that we use only our binaries (which was OK until now). However, as from now, it seems we will need to use also external binaries (java), so the best option is really to prepend the path. For debugging, we can put absolute paths to binaries to debugging output, so that we know which binary is used.